### PR TITLE
fix: docarray pull

### PR DESCRIPTION
**Pull request by azayz**
---

- [ ] This PR references an open issue
- [ ] Test added
- [ ] Documentation adjusted

### CODE REVIEW

File: now/data_loading/data_loading.py [hunk 1]
```diff
@@ -90,13 +82,18 @@ def add_metadata_to_da(da, user_input):
 
 
 def _pull_docarray(dataset_name: str, admin_name: str) -> DocumentArray:
+    dataset_name = (
+        admin_name + '/' + dataset_name if '/' not in dataset_name else dataset_name
+    )
     try:
-        docs = DocumentArray.pull(name=dataset_name, show_progress=True)
+        docs = DocumentArray.pull(
+            name=admin_name + '/' + dataset_name, show_progress=True
```
Comment (joschkabraun) [inline now/data_loading/data_loading.py:90]: don't we prepend `admin_name` two lines above this?

File: now/data_loading/data_loading.py [hunk 2]
```diff
@@ -52,20 +52,12 @@ def load_data(user_input: UserInput, data_class=None) -> DocumentArray:
...
-            dataset_name = 'team-now/' + user_input.dataset_name
-        else:
-            dataset_name = (
-                f'{user_input.admin_name}/{user_input.dataset_name}'
-                if '/' not in user_input.dataset_name
-                else user_input.dataset_name
-            )
-        da = DocumentArray.pull(
-            name=dataset_name, show_progress=True, local_cache=False
+        dataset_name = (
```
Comment (joschkabraun) [inline now/data_loading/data_loading.py:55]: to avoid code duplication, let's use `_pull_docarray` for demo data as well

**Review by joschkabraun — COMMENTED** [review]
Nice cleanup

**Review by IyadhKhalfallah — APPROVED** [review]
LGTM!

File: now/data_loading/data_loading.py [hunk 3]
```diff
@@ -52,20 +52,7 @@ def load_data(user_input: UserInput, data_class=None) -> DocumentArray:
...
-        else:
-            dataset_name = (
-                f'{user_input.admin_name}/{user_input.dataset_name}'
-                if '/' not in user_input.dataset_name
-                else user_input.dataset_name
-            )
-        da = DocumentArray.pull(
-            name=dataset_name, show_progress=True, local_cache=False
-        )
+        da = _pull_docarray(user_input.dataset_name, user_input.admin_name)
```
Comment (joschkabraun) [inline now/data_loading/data_loading.py:55]: we can move it into the other `if` block

**Review by joschkabraun — APPROVED** [review]
LGTM

**Pull request merged**